### PR TITLE
[OPENJDK-575] Adding IBM Semeru 11 CE module and a trial yaml file

### PR DESCRIPTION
**Pull request by levivic**
Hi @jerboaa and @jmtd, as you may already know, there is a strong push from Oracle of switching from IBM Semeru Open Edition to Certified Edition for all the Openj9 related images we release in RH by **the end of March**. Aside from all the business and legal discussions, the management team wants us to validate using the existing RH release proc and toolchains to achieve this transition from pure technical perspective. The process includes building the rpm from srpm (provided by IBM), adding semeru-11-ce modules, and updating all image override files to include the new module/package, like what we did for Semeru open edition.

**I have prepared the semeru-11-certified (only 11 on rhel8 for now) module and I would like to invite you to review it. Since there are some difference between the ce and oe versions. And the module itself is crucial for all other images using it.**

I have also added an override yaml file for this image. I'm sure there could be more changes coming, i.e, the downstream repos/branches, image names, signing intent, etc., when a final decision is made about this transition. So for those changes I would be less concerned at this moment.

Thanks in advance for any feedback! 

https://issues.redhat.com/browse/OPENJDK-575
Signed-off-by: Lei Zhang <lzhan@redhat.com>

- [x] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: semeru-ce-11-rhel8.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- name: "io.k8s.display-name"
+  value: "Java Applications"
+- name: "io.openshift.tags"
+  value: "builder,java"
+- name: "maintainer"
+  value: "Red Hat OpenJDK <openjdk@redhat.com>"
+- name: "com.redhat.component"
+  value: "openj9-11-rhel8-container"
+- name: "usage"
+  value: "https://access.redhat.com/documentation/en-us/red_hat_jboss_middleware_for_openshift/3/html/red_hat_java_s2i_for_openshift/"
```
Comment (jerboaa) [inline semeru-ce-11-rhel8.yaml]: This should probably get updated to: https://access.redhat.com/documentation/en-us/openjdk/11/html/using_openjdk_11_source-to-image_for_openshift/index

File: semeru-ce-11-rhel8.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+ports:
+- value: 8080
+- value: 8443
+
+packages:
+  manager: microdnf
+  content_sets:
+    x86_64:
+    - rhel-8-for-x86_64-baseos-rpms
+    - rhel-8-for-x86_64-appstream-rpms
```
Comment (jerboaa) [inline semeru-ce-11-rhel8.yaml]: Why content sets for `x86_64` and `aarch64`? I thought this would be ppc64le and s390x only.

**Review by jerboaa — APPROVED** [review]
Seems fine. Just a few minor comments.

**Review by jerboaa — APPROVED** [review]

**Comment by levivic** [comment]
@jerboaa thanks for the comments!

**Review by jmtd — APPROVED** [review]

**Comment by jmtd** [comment]
I've kicked off the GItHub CI jobs for this PR, I have no reason to think that this PR should influence or cause problems for the other images, but we might as well wait for the green lights before merging.

**Pull request merged**